`config export image` doesn't support all characters

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

The character `nf-md-warehouse`/`\udb83\udf81`/󰾁 isn't rendered correctly.

![nf-md-warehouse](https://github.com/JanDeDobbeleer/oh-my-posh/assets/60285058/3852c054-c29f-4a02-8308-70f55f0995fb)
<https://www.nerdfonts.com/cheat-sheet>

#### Export
Command: `oh-my-posh config export image --cursor-padding 50`
(the char is after `master`)
![export](https://github.com/JanDeDobbeleer/oh-my-posh/assets/60285058/021a1a52-5f13-43b2-94ae-c7f4dfc9d75b)
#### Screenshot
![screenshot](https://github.com/JanDeDobbeleer/oh-my-posh/assets/60285058/832dbe9d-0b0d-4c58-8595-58cb47ee0a7d)


### Theme

#### git part
```
        {
          "type": "git",
          "style": "plain",
          "foreground": "#C678DD",
          "properties": {
            "fetch_status": true,
            "fetch_version": true
          },
          "template": "{{ .HEAD }} {{if .BranchStatus }}{{ .BranchStatus }} {{ end }}{{if .Upstream }}{{ .Upstream }} {{ end }}<#66CCFF>{{if .StashCount }}\udb83\udf81 {{ .StashCount }} {{ end }}</><#DD8822>{{ if .Working.Changed }}\uf044 {{ .Working.String }} {{ end }}</><#77DD65>{{ if .Staging.Changed }}\uf046 {{ .Staging.String }} {{ end }}</>"
        }
```
#### entire
```
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "blocks": [
    {
      "alignment": "left",
      "segments": [
        {
          "type": "root",
          "style": "plain",
          "foreground": "#E06C75",
          "template": "root "
        },
        {
          "type": "path",
          "style": "plain",
          "foreground": "#55B9C4",
          "properties": {
            "style": "folder"
          },
          "template": "\ueaf7 {{ .Path }} "
        },
        {
          "type": "project",
          "style": "plain",
          "foreground": "#ccbb22",
          "template": "{{ if .Error }}{{ .Error }}{{ else }}{{ if .Version }} {{.Version}}{{ end }} {{ if .Name }}{{ .Name }}{{ end }}{{ end }} "
        },
        {
          "type": "git",
          "style": "plain",
          "foreground": "#C678DD",
          "properties": {
            "fetch_status": true,
            "fetch_version": true
          },
          "template": "{{ .HEAD }} {{if .BranchStatus }}{{ .BranchStatus }} {{ end }}{{if .Upstream }}{{ .Upstream }} {{ end }}<#66CCFF>{{if .StashCount }}\udb83\udf81 {{ .StashCount }} {{ end }}</><#DD8822>{{ if .Working.Changed }}\uf044 {{ .Working.String }} {{ end }}</><#77DD65>{{ if .Staging.Changed }}\uf046 {{ .Staging.String }} {{ end }}</>"
        }
      ],
      "type": "prompt"
    },
    {
      "alignment": "right",
      "segments": [
        {
          "type": "sysinfo",
          "style": "plain",
          "foreground": "#8f43f3",
          "template": " \ue266 {{ (div ((sub .PhysicalTotalMemory .PhysicalFreeMemory)|float64) 1000000000.0) }}/{{ (div .PhysicalTotalMemory 1000000000.0) }}GB",
          "properties": {
            "precision": 2
          }
        },
        {
          "type": "battery",
          "style": "plain",
          "foreground": "#193549",
          "foreground_templates": [
            "{{if eq \"Charging\" .State.String}}#40c4ff{{end}}",
            "{{if eq \"Discharging\" .State.String}}#ff5722{{end}}",
            "{{if eq \"Full\" .State.String}}#4caf50{{end}}"
          ],
          "template": " {{ if not .Error }}{{ .Icon }}{{ .Percentage }}{{ end }}",
          "properties": {
            "discharging_icon": " ",
            "charging_icon": " ",
            "charged_icon": " "
          }
        },
        {
          "type": "connection",
          "style": "plain",
          "foreground": "white",
          "template": " {{ if eq .Type \"wifi\"}}\uf1eb wifi{{ else if eq .Type \"ethernet\"}}\ueba9 ethernet{{ end }}"
        },
        {
          "type": "docker",
          "style": "plain",
          "foreground": "#0B59E7",
          "template": " \uf308 {{ .Context }}"
        },
        {
          "type": "java",
          "style": "plain",
          "foreground": "#ff4500",
          "template": " \ue738 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }}"
        },
        {
          "type": "kotlin",
          "style": "plain",
          "foreground": "#906cff",
          "template": " \ue634 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }}"
        },
        {
          "type": "lua",
          "style": "plain",
          "foreground": "blue",
          "template": " \ue620 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }}"
        },
        {
          "type": "python",
          "style": "plain",
          "foreground": "#4682b4",
          "template": " \ue235 {{ if .Error }}{{ .Error }}{{ else }}{{ if .Venv }}{{ .Venv }} {{ end }}{{ .Full }}{{ end }}"
        },
        {
          "type": "rust",
          "style": "plain",
          "foreground": "#9a7b4f",
          "template": " \ue7a8 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }}"
        },
        {
          "type": "node",
          "style": "plain",
          "foreground": "#98C379",
          "template": " \udb80\udf99 {{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Full }}"
        }
      ],
      "type": "prompt"
    },
    {
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "type": "ytm",
          "style": "plain",
          "foreground": "#ff0000",
          "properties": {
            "playing_icon": "\uf04b ",
            "paused_icon": "\uf04c ",
            "stopped_icon": "\uf04d "
          },
          "template": "\uf167 {{ .Icon }}{{ if ne .Status \"stopped\" }}{{ .Artist }} - {{ .Track }}{{ end }}"
        },
        {
          "type": "spotify",
          "style": "plain",
          "foreground": "#1eda61",
          "properties": {
            "playing_icon": "\uf04b ",
            "paused_icon": "\uf04c ",
            "stopped_icon": "\uf04d "
          },
          "template": "\uf1bc {{ .Icon }}{{ if ne .Status \"stopped\" }}{{ .Artist }} - {{ .Track }}{{ end }}"
        }
      ],
      "type": "prompt"
    },
    {
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "type": "executiontime",
          "foreground": "#be9ddf",
          "properties": {
            "style": "dallas",
            "threshold": 0
          },
          "style": "diamond",
          "template": "<#ffffff>\uf252</> {{ .FormattedMs }}s "
        },
        {
          "type": "status",
          "style": "plain",
          "properties": {
            "always_enabled": true
          },
          "template": "{{ if .Error }}<#ff5522>x</>{{ else }}<#63F08C>\u279c</>{{end}} "
        }
      ],
      "type": "prompt"
    }
  ],
  "version": 2
}
```

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 18.26.1

Shell: pwsh (7.4.0)

Prompt:

 rust-image-serialport  0.1.0 rust-image-serialport master ≢ 󰾁 1  ?5                     9/16GB  99  wifi  1.73.0
 0s ➜

Segments:

ConsoleTitle(false)                        -   0 ms
Root(false)                                -   0 ms
Path(true)                                 -   2 ms
Project(true)                              -   2 ms
Git(true)                                  -  53 ms
Sysinfo(true)                              -   0 ms
Battery(true)                              -  20 ms
Connection(true)                           -   7 ms
Docker(false)                              -   0 ms
Java(false)                                -   5 ms
Kotlin(false)                              -   2 ms
Lua(false)                                 -   2 ms
Python(false)                              -  52 ms
Rust(true)                                 - 122 ms
Node(false)                                -   5 ms
Ytm(false)                                 -   1 ms
Spotify(false)                             -   6 ms
Executiontime(true)                        -   0 ms
Status(true)                               -   0 ms

Run duration: 198.2862ms

Cache path: C:\Users\26523\AppData\Local\oh-my-posh

Config path: C:\Users\26523\Documents\PowerShell\Scripts\star.omp.json

Logs:

[DEBUG] 18:02:08.816 shell.go:Getenv:398 → NO DATA
[TRACE] 18:02:08.816 shell.go:Getenv(TERM_PROGRAM) - 0s
[DEBUG] 18:02:08.816 shell.go:Getenv:398 → C:\Users\26523\AppData\Local
[TRACE] 18:02:08.816 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 18:02:08.816 shell_windows.go:CachePath() - 588µs
[DEBUG] 18:02:08.818 shell.go:Getenv:398 → C:\Users\26523\Documents\PowerShell\Scripts\star.omp.json
[TRACE] 18:02:08.818 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 18:02:08.818 shell.go:Shell:652 → no shell name provided in flags, trying to detect it
[DEBUG] 18:02:08.822 shell.go:Shell:660 → process name: pwsh.exe
[TRACE] 18:02:08.822 shell.go:Shell() - 4.576ms
[TRACE] 18:02:08.822 shell.go:resolveConfigPath() - 4.576ms
[TRACE] 18:02:08.823 shell.go:Init() - 6.8351ms
[TRACE] 18:02:08.823 shell.go:Flags() - 0s
[TRACE] 18:02:08.824 config.go:loadConfig() - 1.1042ms
[TRACE] 18:02:08.824 shell.go:Flags() - 0s
[DEBUG] 18:02:08.824 shell.go:Getenv:398 → NO DATA
[TRACE] 18:02:08.824 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 18:02:08.824 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.824 shell_windows.go:WindowsRegistryKeyValue:215 → ColorizationColor(DWORD): 0xC40063B1
[TRACE] 18:02:08.824 shell.go:Shell() - 0s
[DEBUG] 18:02:08.824 shell.go:Getenv:398 → 7.4.0
[TRACE] 18:02:08.824 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 18:02:08.824 debug.go:PrintDebug:22 → Segment: Title
[DEBUG] 18:02:08.824 text.go:Render:70 → Rendering template:
[DEBUG] 18:02:08.824 shell.go:Getenv:398 → 47
[TRACE] 18:02:08.824 shell.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 18:02:08.824 shell.go:Getenv:398 → 1
[TRACE] 18:02:08.824 shell.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 18:02:08.824 shell.go:Flags() - 0s
[TRACE] 18:02:08.824 shell.go:Flags() - 0s
[TRACE] 18:02:08.824 shell.go:Shell() - 0s
[TRACE] 18:02:08.824 shell.go:Shell() - 0s
[TRACE] 18:02:08.824 shell.go:Flags() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:Flags() - 0s
[TRACE] 18:02:08.824 shell.go:Flags() - 0s
[DEBUG] 18:02:08.824 shell.go:Pwd:428 → C:\Users\26523\Documents\Github\rust-image-serialport
[TRACE] 18:02:08.824 shell.go:Pwd() - 0s
[TRACE] 18:02:08.824 shell.go:Pwd() - 0s
[TRACE] 18:02:08.824 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.824 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.824 segment.go:SetEnabled:512 → Segment: Git
[DEBUG] 18:02:08.824 shell_windows.go:56 → C:\Users\26523
[DEBUG] 18:02:08.824 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.824 segment.go:SetEnabled:512 → Segment: Path
[DEBUG] 18:02:08.824 segment.go:SetEnabled:512 → Segment: Root
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:Pwd() - 0s
[TRACE] 18:02:08.824 shell.go:Shell() - 0s
[TRACE] 18:02:08.824 shell.go:Flags() - 0s
[TRACE] 18:02:08.824 shell_windows.go:Root() - 0s
[DEBUG] 18:02:08.824 properties.go:GetBool:22 → mapped_locations_enabled: true
[TRACE] 18:02:08.824 shell.go:Flags() - 0s
[DEBUG] 18:02:08.824 properties.go:GetString:28 → 
[DEBUG] 18:02:08.824 properties.go:GetString:28 → 
[DEBUG] 18:02:08.824 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.824 properties.go:GetString:28 → ~
[DEBUG] 18:02:08.824 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 18:02:08.824 shell.go:PathSeparator() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[TRACE] 18:02:08.824 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.824 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.824 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.824 segment.go:SetEnabled:512 → Segment: Project
[TRACE] 18:02:08.824 shell.go:Pwd() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:PathSeparator() - 0s
[TRACE] 18:02:08.825 shell.go:PathSeparator() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:PathSeparator() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:PathSeparator() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:PathSeparator() - 0s
[TRACE] 18:02:08.825 shell.go:PathSeparator() - 0s
[DEBUG] 18:02:08.825 properties.go:GetString:28 → folder
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:GOOS() - 0s
[TRACE] 18:02:08.825 shell.go:PathSeparator() - 0s
[DEBUG] 18:02:08.825 properties.go:GetString:28 → NO DATA
[TRACE] 18:02:08.825 shell.go:PathSeparator() - 0s
[DEBUG] 18:02:08.825 properties.go:GetString:28 → \
[TRACE] 18:02:08.825 shell.go:Pwd() - 0s
[TRACE] 18:02:08.825 shell_windows.go:IsWsl() - 0s
[TRACE] 18:02:08.825 shell.go:StackCount() - 0s
[DEBUG] 18:02:08.825 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.825 shell.go:HasFiles(package.json) - 538µs
[TRACE] 18:02:08.825 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.825 shell.go:HasFiles:459 → true
[TRACE] 18:02:08.825 shell.go:HasFiles(Cargo.toml) - 524.6µs
[TRACE] 18:02:08.825 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.826 shell.go:FileContent:537 ↓
    [package]
    name = "rust-image-serialport"
    version = "0.1.0"
    edition = "2021"

    # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

    [dependencies]
    serialport = { version = "4.2.2", features = ["usbportinfo-interface"] }
    clap = { version = "4.4.6", features = ["derive"] }
    rustbitmap = "0.1.0"
    reqwest = { version = "0.11", features = ["blocking", "json"] }
    image = "0.24.7"

[TRACE] 18:02:08.826 shell.go:FileContent(Cargo.toml) - 569.7µs
[TRACE] 18:02:08.826 shell_windows.go:Root() - 0s
[TRACE] 18:02:08.826 shell.go:Shell() - 0s
[TRACE] 18:02:08.826 shell.go:StatusCodes() - 0s
[TRACE] 18:02:08.826 shell_windows.go:IsWsl() - 0s
[DEBUG] 18:02:08.826 shell.go:TemplateCache:833 → environment: [ALLUSERSPROFILE=C:\ProgramData APPDATA=C:\Users\26523\AppData\Roaming ARMCC=C:\Keil_v5\ARM\ARMCC chocolateyinstall=C:\Users\26523\AppData\Local\Programs\WingetUI\choco-cli CODELLDB=C:\Users\26523\Software\codelldb\codelldb_adapter\adapter CommonProgramFiles=C:\Program Files\Common Files CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files CommonProgramW6432=C:\Program Files\Common Files COMPUTERNAME=LAPTOP-RIUHQE37 ComSpec=C:\WINDOWS\system32\cmd.exe CONDA_PROMPT_MODIFIER=False CPLUS_INCLUDE_PATH=C:\Users\26523\scoop\apps\gcc\current\include C_INCLUDE_PATH=C:\Users\26523\scoop\apps\gcc\current\include DriverData=C:\Windows\System32\Drivers\DriverData EIDE_TOOLS_GCC_ARM=C:\Users\26523\.eide\tools\gcc_arm GOPATH=C:\Users\26523\go HOMEDRIVE=C: HOMEPATH=\Users\26523 LOCALAPPDATA=C:\Users\26523\AppData\Local LOCAL_PROGRAMS=C:\Users\26523\AppData\Local\Programs LOGONSERVER=\\LAPTOP-RIUHQE37 LUA_CPATH=C:\Users\26523\scoop\apps\lua\current LUA_EXE_PATH=C:\Users\26523\scoop\apps\lua\current MSSTORE_PYTHON_SCRIPT=C:\Users\26523\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\Scripts NUMBER_OF_PROCESSORS=8 NVM_HOME=C:\Users\26523\AppData\Roaming\nvm NVM_SYMLINK=C:\Program Files\nodejs OneDrive=C:\Users\26523\OneDrive OneDriveConsumer=C:\Users\26523\OneDrive OS=Windows_NT Path=C:\Program Files\PowerShell\7;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Git\cmd;C:\Users\26523\AppData\Roaming\nvm;C:\Program Files\nodejs;C:\Program Files\Go\bin;C:\Program Files\Neovide\;C:\Program Files\CMake\bin;C:\Program Files\dotnet\;C:\Program Files\GitHub CLI\;C:\Program Files\gsudo\Current;C:\Program Files (x86)\STMicroelectronics\STM32 ST-LINK Utility\ST-LINK Utility;C:\Program Files\PowerShell\7\;C:\Users\26523\scoop\apps\gpg\current\bin;C:\Users\26523\scoop\apps\gcc\current\bin;C:\Users\26523\scoop\shims;C:\Users\26523\.cargo\bin;C:\Users\26523\go\bin;C:\Users\26523\.local\bin;C:\Users\26523\.dotnet\tools;C:\Users\26523\AppData\Local\Programs\Python\Python311\;C:\Users\26523\AppData\Local\Programs\Python\Python311\Scripts\;C:\Users\26523\AppData\Local\Programs\WingetUI\choco-cli\bin;C:\Users\26523\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\26523\AppData\Local\Programs\oh-my-posh\bin;C:\Users\26523\AppData\Local\Microsoft\WinGet\Packages\Barry-ran.QtScrcpy_;C:\Users\26523\AppData\Local\Microsoft\WinGet\Packages\Genymobile.scrcpy_Microsoft.Winget.Source_8wekyb3d8bbwe\scrcpy-win64-v2.1.1;C:\Users\26523\AppData\Local\Microsoft\WinGet\Packages\Gyan.FFmpeg_Microsoft.Winget.Source_8wekyb3d8bbwe\ffmpeg-6.0-full_build\bin;C:\Users\26523\AppData\Local\Microsoft\WinGet\Packages\Helix.Helix_Microsoft.Winget.Source_8wekyb3d8bbwe\helix-23.10-x86_64-windows;C:\Users\26523\AppData\Local\Microsoft\WinGet\Packages\Ninja-build.Ninja_Microsoft.Winget.Source_8wekyb3d8bbwe;C:\Users\26523\AppData\Local\Microsoft\WinGet\Packages\gsass1.NTop_Microsoft.Winget.Source_8wekyb3d8bbwe;C:\Users\26523\AppData\Local\Microsoft\WinGet\Packages\jftuga.less_Microsoft.Winget.Source_8wekyb3d8bbwe;C:\Users\26523\AppData\Local\Microsoft\WinGet\Packages\junegunn.fzf_Microsoft.Winget.Source_8wekyb3d8bbwe;C:\Users\26523\AppData\Local\Microsoft\WinGet\Packages\sharkdp.bat_Microsoft.Winget.Source_8wekyb3d8bbwe\bat-v0.23.0-x86_64-pc-windows-msvc;C:\Program Files\nodejs;C:\Program Files\LLVM\bin;C:\Program Files\Neovim\bin;C:\Users\26523\AppData\Local\Pandoc\;C:\Users\26523\AppData\Roaming\nvm;C:\Users\26523\AppData\Local\Microsoft\WindowsApps;C:\Users\26523\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\Scripts;C:\Users\26523\AppData\Roaming\python\python311\scripts;C:\Users\26523\Documents\PowerShell\Scripts;C:\Keil_v5\ARM\ARMCC\bin;C:\Users\26523\Software\codelldb\codelldb_adapter\adapter;C:\Users\26523\.eide\tools\gcc_arm\bin;C:\Program Files\qemu;;C:\Users\26523\AppData\Local\Programs\oh-my-posh\bin PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.CPL POSH_AZURE_ENABLED=False POSH_CURSOR_COLUMN=1 POSH_CURSOR_LINE=47 POSH_GIT_ENABLED=False POSH_INSTALLER=winget POSH_PID=6088 POSH_SHELL_VERSION=7.4.0 POSH_THEME=C:\Users\26523\Documents\PowerShell\Scripts\star.omp.json POSH_THEMES_PATH=C:\Users\26523\AppData\Local\Programs\oh-my-posh\themes POWERLINE_COMMAND=oh-my-posh PowerShellScripts=C:\Users\26523\Documents\PowerShell\Scripts POWERSHELL_DISTRIBUTION_CHANNEL=MSI:Windows 10 Home China POWERSHELL_TELEMETRY_OPTOUT=1 PROCESSOR_ARCHITECTURE=AMD64 PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 140 Stepping 1, GenuineIntel PROCESSOR_LEVEL=6 PROCESSOR_REVISION=8c01 ProgramData=C:\ProgramData ProgramFiles=C:\Program Files ProgramFiles(x86)=C:\Program Files (x86) ProgramW6432=C:\Program Files PSModulePath=C:\Users\26523\Documents\PowerShell\Modules;C:\Program Files\PowerShell\Modules;c:\program files\powershell\7\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules PUBLIC=C:\Users\Public PWD=C:\Users\26523\Pictures PYTHONIOENCODING=utf-8 QEMU=C:\Program Files\qemu RUSTUP_DIST_SERVER=https://mirrors.ustc.edu.cn/rust-static RUSTUP_UPDATE_ROOT=https://mirrors.ustc.edu.cn/rust-static/rustup SESSIONNAME=Console SystemDrive=C: SystemRoot=C:\WINDOWS TEMP=C:\Users\26523\AppData\Local\Temp TMP=C:\Users\26523\AppData\Local\Temp USERDOMAIN=LAPTOP-RIUHQE37 USERDOMAIN_ROAMINGPROFILE=LAPTOP-RIUHQE37 USERNAME=26523 USERPROFILE=C:\Users\26523 VIRTUAL_ENV_DISABLE_PROMPT=1 windir=C:\WINDOWS WINGET_PACKAGES=C:\Users\26523\AppData\Local\Microsoft\WinGet\Packages WSLENV=WT_SESSION:WT_PROFILE_ID: WT_PROFILE_ID={574e775e-4f2a-5b96-ac1e-a2962a402336} WT_SESSION=39739b7c-fcc8-448e-972d-ce2dd512175f ZES_ENABLE_SYSMAN=1 _ZL_ADD_ONCE=1 _ZL_MATCH_MODE=1 _ZL_RANDOM=449179975]
[TRACE] 18:02:08.826 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.826 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[TRACE] 18:02:08.826 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.826 shell.go:User:563 → 26523
[TRACE] 18:02:08.826 shell.go:User() - 0s
[DEBUG] 18:02:08.827 shell.go:Host:575 → LAPTOP-RIUHQE37
[TRACE] 18:02:08.827 shell.go:Host() - 510.2µs
[TRACE] 18:02:08.827 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.827 shell.go:Getenv:398 → NO DATA
[TRACE] 18:02:08.827 shell.go:Getenv(SHLVL) - 0s
[TRACE] 18:02:08.827 shell.go:TemplateCache() - 510.2µs
[DEBUG] 18:02:08.827 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 18:02:08.827 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 18:02:08.827 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 18:02:08.827 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 18:02:08.827 shell_windows.go:DirIsWritable() - 1.6406ms
[TRACE] 18:02:08.827 shell.go:TemplateCache() - 0s
[DEBUG] 18:02:08.834 shell.go:CommandPath:614 → C:\Program Files\Git\cmd\git.exe
[TRACE] 18:02:08.834 shell.go:CommandPath(git.exe) - 10.5988ms
[TRACE] 18:02:08.834 shell.go:HasCommand(git.exe) - 10.5988ms
[TRACE] 18:02:08.834 shell.go:Pwd() - 0s
[TRACE] 18:02:08.834 shell.go:HasParentFilePath(.git) - 0s
[DEBUG] 18:02:08.834 properties.go:GetStringArray:52 → exclude_folders: []
[DEBUG] 18:02:08.834 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.834 properties.go:GetBool:22 → fetch_user: false
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.834 shell.go:Getenv:398 → NO DATA
[TRACE] 18:02:08.834 shell.go:Getenv(POSH_GIT_STATUS) - 0s
[DEBUG] 18:02:08.834 properties.go:GetBool:22 → fetch_status: true
[DEBUG] 18:02:08.834 properties.go:GetStringArray:52 → ignore_status: []
[DEBUG] 18:02:08.834 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.834 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.834 properties.go:GetKeyValueMap:46 → status_formats: map[]
[DEBUG] 18:02:08.834 properties.go:GetKeyValueMap:46 → untracked_modes: map[]
[DEBUG] 18:02:08.834 properties.go:GetKeyValueMap:46 → ignore_submodules: map[]
[DEBUG] 18:02:08.877 shell.go:RunCommand:593 ↓
    # branch.oid 307679ccc893acdaa479876fef3119fe704bc640
    # branch.head master
    ? .gitignore
    ? .vscode/
    ? Cargo.toml
    ? src/
    ? star.png
[TRACE] 18:02:08.877 shell.go:RunCommand(git.exe -C C:/Users/26523/Documents/Github/rust-image-serialport --no-optional-locks -c core.quotepath=false -c color.status=false status -unormal --branch --porcelain=2) - 42.8842ms
[DEBUG] 18:02:08.877 properties.go:GetString:28 → 
[DEBUG] 18:02:08.877 properties.go:GetInt:40 → branch_max_length: 0
[DEBUG] 18:02:08.877 shell.go:HasFolder:507 → false
[TRACE] 18:02:08.877 shell.go:HasFolder(C:\Users\26523\Documents\Github\rust-image-serialport\.git/rebase-merge) - 139.6µs
[DEBUG] 18:02:08.877 shell.go:HasFolder:507 → false
[TRACE] 18:02:08.877 shell.go:HasFolder(C:\Users\26523\Documents\Github\rust-image-serialport\.git/rebase-apply) - 0s
[DEBUG] 18:02:08.877 properties.go:GetString:28 → 
[DEBUG] 18:02:08.877 shell.go:HasFilesInDir:478 → false
[TRACE] 18:02:08.877 shell.go:HasFilesInDir(MERGE_MSG) - 0s
[DEBUG] 18:02:08.877 shell.go:HasFilesInDir:478 → false
[TRACE] 18:02:08.877 shell.go:HasFilesInDir(CHERRY_PICK_HEAD) - 0s
[DEBUG] 18:02:08.877 shell.go:HasFilesInDir:478 → false
[TRACE] 18:02:08.877 shell.go:HasFilesInDir(REVERT_HEAD) - 0s
[DEBUG] 18:02:08.877 shell.go:HasFilesInDir:478 → false
[TRACE] 18:02:08.877 shell.go:HasFilesInDir(sequencer/todo) - 0s
[DEBUG] 18:02:08.877 properties.go:GetString:28 → ≢
[DEBUG] 18:02:08.877 properties.go:GetBool:22 → fetch_upstream_icon: false
[TRACE] 18:02:08.877 shell.go:TemplateCache() - 0s
[DEBUG] 18:02:08.878 text.go:Render:70 → Rendering template: {{ .HEAD }} {{if .BranchStatus }}{{ .BranchStatus }} {{ end }}{{if .Upstream }}{{ .Upstream }} {{ end }}<#66CCFF>{{if .StashCount }}󰾁 {{ .StashCount }} {{ end }}</><#DD8822>{{ if .Working.Changed }} {{ .Working.String }} {{ end }}</><#77DD65>{{ if .Staging.Changed }} {{ .Staging.String }} {{ end }}</>
[DEBUG] 18:02:08.878 text.go:Render:70 → Rendering template:  {{ .Path }}
[DEBUG] 18:02:08.878 text.go:Render:70 → Rendering template: {{ if .Error }}{{ .Error }}{{ else }}{{ if .Version }} {{.Version}}{{ end }} {{ if .Name }}{{ .Name }}{{ end }}{{ end }}
[TRACE] 18:02:08.879 shell.go:TemplateCache() - 0s
[TRACE] 18:02:08.879 shell.go:Shell() - 0s
[TRACE] 18:02:08.879 shell.go:TemplateCache() - 0s
[TRACE] 18:02:08.879 shell.go:TemplateCache() - 0s
[TRACE] 18:02:08.879 shell.go:Shell() - 0s
[DEBUG] 18:02:08.879 shell.go:FileContent:537 ↓
    0000000000000000000000000000000000000000 bf113a2ddfffef21fb6644ce5c7135a757658d9d CAESIUS-TIM <60285058+CAESIUS-TIM@users.noreply.github.com> 1700647305 +0800        WIP on master: 307679c Init

[TRACE] 18:02:08.879 shell.go:FileContent(C:\Users\26523\Documents\Github\rust-image-serialport\.git/logs/refs/stash) - 0s
[TRACE] 18:02:08.879 shell.go:Shell() - 0s
[DEBUG] 18:02:08.879 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:08.879 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:08.879 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:08.879 text.go:Render:70 → Rendering template: plain
[TRACE] 18:02:08.879 shell.go:Shell() - 0s
[TRACE] 18:02:08.879 shell.go:Shell() - 0s
[TRACE] 18:02:08.879 shell.go:Shell() - 0s
[TRACE] 18:02:08.879 shell.go:Flags() - 0s
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.880 shell.go:GOOS() - 0s
[TRACE] 18:02:08.880 shell.go:Flags() - 0s
[DEBUG] 18:02:08.880 segment.go:SetEnabled:512 → Segment: Node
[TRACE] 18:02:08.880 shell.go:Flags() - 0s
[DEBUG] 18:02:08.880 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .npmrc .vue]
[DEBUG] 18:02:08.880 properties.go:GetStringArray:52 → folders: []
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 properties.go:GetBool:22 → home_enabled: false
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[DEBUG] 18:02:08.880 shell.go:Getenv:398 → NO DATA
[DEBUG] 18:02:08.880 properties.go:GetString:28 → files
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.880 shell.go:Getenv(JAVA_HOME) - 0s
[TRACE] 18:02:08.880 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.880 properties.go:GetBool:22 → fetch_package_manager: false
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 segment.go:SetEnabled:512 → Segment: Battery
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.880 shell.go:GOOS() - 0s
[TRACE] 18:02:08.880 shell_windows.go:IsWsl() - 0s
[DEBUG] 18:02:08.880 segment.go:SetEnabled:512 → Segment: Java
[DEBUG] 18:02:08.880 properties.go:GetStringArray:52 → extensions: [pom.xml build.gradle.kts build.sbt .java-version .deps.edn project.clj build.boot *.java *.class *.gradle *.jar *.clj *.cljc]
[DEBUG] 18:02:08.880 properties.go:GetStringArray:52 → folders: []
[DEBUG] 18:02:08.880 properties.go:GetBool:22 → home_enabled: false
[TRACE] 18:02:08.880 shell.go:Flags() - 0s
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[TRACE] 18:02:08.880 shell.go:Flags() - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[DEBUG] 18:02:08.880 properties.go:GetString:28 → files
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[TRACE] 18:02:08.880 shell.go:Flags() - 0s
[DEBUG] 18:02:08.880 properties.go:GetString:28 → lua
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[TRACE] 18:02:08.880 shell.go:Flags() - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.880 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.880 shell.go:Flags() - 0s
[TRACE] 18:02:08.880 shell.go:GOOS() - 0s
[TRACE] 18:02:08.880 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.880 segment.go:SetEnabled:512 → Segment: Connection
[DEBUG] 18:02:08.880 segment.go:SetEnabled:512 → Segment: Docker
[DEBUG] 18:02:08.880 segment.go:SetEnabled:512 → Segment: Lua
[DEBUG] 18:02:08.880 shell.go:Getenv:398 → NO DATA
[DEBUG] 18:02:08.880 properties.go:GetString:28 → environment
[DEBUG] 18:02:08.880 properties.go:GetString:28 → wifi|ethernet
[DEBUG] 18:02:08.880 properties.go:GetStringArray:52 → extensions: [*.lua *.rockspec]
[DEBUG] 18:02:08.880 properties.go:GetInt:40 → precision: 2
[TRACE] 18:02:08.880 shell.go:Getenv(DOCKER_MACHINE_NAME) - 0s
[DEBUG] 18:02:08.880 properties.go:GetStringArray:52 → folders: [lua]
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell.go:Getenv:398 → NO DATA
[DEBUG] 18:02:08.880 properties.go:GetBool:22 → home_enabled: false
[TRACE] 18:02:08.880 shell.go:Getenv(DOCKER_HOST) - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell.go:Getenv:398 → NO DATA
[TRACE] 18:02:08.880 shell.go:GOOS() - 0s
[TRACE] 18:02:08.880 shell.go:Getenv(DOCKER_CONTEXT) - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[DEBUG] 18:02:08.880 segment.go:SetEnabled:512 → Segment: Python
[DEBUG] 18:02:08.880 properties.go:GetString:28 → files
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 18:02:08.880 shell.go:Getenv:398 → NO DATA
[DEBUG] 18:02:08.880 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[TRACE] 18:02:08.880 shell.go:Getenv(DOCKER_CONFIG) - 0s
[DEBUG] 18:02:08.880 properties.go:GetBool:22 → home_enabled: false
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[DEBUG] 18:02:08.880 properties.go:GetBool:22 → fetch_virtual_env: true
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.880 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.880 segment.go:SetEnabled:512 → Segment: Sysinfo
[TRACE] 18:02:08.880 shell.go:TemplateCache() - 0s
[ERROR] 18:02:08.880 shell.go:FileContent:533 → open C:\Users\26523\.docker\config.json: The system cannot find the path specified.
[TRACE] 18:02:08.880 shell.go:FileContent(C:\Users\26523\.docker\config.json) - 505.5µs
[TRACE] 18:02:08.880 shell.go:Flags() - 0s
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.880 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.880 segment.go:SetEnabled:512 → Segment: Kotlin
[DEBUG] 18:02:08.880 properties.go:GetStringArray:52 → extensions: [*.kt *.kts *.ktm]
[DEBUG] 18:02:08.880 properties.go:GetStringArray:52 → folders: []
[DEBUG] 18:02:08.880 properties.go:GetBool:22 → home_enabled: false
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell_windows.go:56 → C:\Users\26523
[DEBUG] 18:02:08.880 properties.go:GetString:28 → files
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.880 shell.go:HasFiles(*.js) - 536.5µs
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.880 shell.go:HasFiles(pom.xml) - 536.5µs
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.880 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.880 shell.go:HasFiles(*.lua) - 536.5µs
[TRACE] 18:02:08.880 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.881 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.881 shell.go:HasFiles(*.kt) - 569.9µs
[DEBUG] 18:02:08.881 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.881 shell.go:Pwd() - 0s
[TRACE] 18:02:08.881 shell.go:HasFiles(build.gradle.kts) - 569.9µs
[TRACE] 18:02:08.881 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.881 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.881 shell.go:HasFiles(*.ts) - 569.9µs
[TRACE] 18:02:08.881 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.881 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.881 shell.go:HasFiles(*.rockspec) - 569.9µs
[DEBUG] 18:02:08.881 shell.go:HasFolder:507 → false
[TRACE] 18:02:08.882 shell.go:Flags() - 0s
[TRACE] 18:02:08.882 shell.go:Pwd() - 0s
[TRACE] 18:02:08.882 shell.go:Pwd() - 0s
[TRACE] 18:02:08.882 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.882 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.882 shell.go:HasFiles(package.json) - 1.0432ms
[DEBUG] 18:02:08.882 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:08.882 shell.go:Pwd() - 0s
[TRACE] 18:02:08.882 shell.go:GOOS() - 0s
[DEBUG] 18:02:08.882 segment.go:SetEnabled:512 → Segment: Rust
[DEBUG] 18:02:08.882 properties.go:GetStringArray:52 → extensions: [*.rs Cargo.toml Cargo.lock]
[DEBUG] 18:02:08.882 properties.go:GetStringArray:52 → folders: []
[DEBUG] 18:02:08.882 properties.go:GetBool:22 → home_enabled: false
[TRACE] 18:02:08.882 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.882 shell_windows.go:56 → C:\Users\26523
[DEBUG] 18:02:08.882 properties.go:GetString:28 → files
[TRACE] 18:02:08.882 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.882 shell.go:HasFiles:464 → false
[DEBUG] 18:02:08.882 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.882 shell.go:HasFiles(*.ktm) - 538.2µs
[TRACE] 18:02:08.882 shell.go:HasFiles(.java-version) - 538.2µs
[TRACE] 18:02:08.882 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.882 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.882 shell.go:HasFiles(.nvmrc) - 538.2µs
[TRACE] 18:02:08.882 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.882 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.882 shell.go:HasFiles(*.rs) - 538.2µs
[TRACE] 18:02:08.882 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.882 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.882 shell.go:HasFiles(.deps.edn) - 0s
[TRACE] 18:02:08.882 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.883 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.883 shell.go:HasFiles(pnpm-workspace.yaml) - 522.5µs
[TRACE] 18:02:08.883 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.883 shell.go:HasFiles:459 → true
[TRACE] 18:02:08.883 shell.go:HasFiles(Cargo.toml) - 522.5µs
[DEBUG] 18:02:08.883 properties.go:GetBool:22 → fetch_version: true
[DEBUG] 18:02:08.883 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.883 shell.go:HasFiles(project.clj) - 522.5µs
[TRACE] 18:02:08.883 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.883 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.883 shell.go:HasFiles(.pnpmfile.cjs) - 518µs
[TRACE] 18:02:08.883 shell.go:Pwd() - 0s
[TRACE] 18:02:08.883 networks_windows.go:wifiNetwork() - 0s
[DEBUG] 18:02:08.884 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.884 shell.go:HasFiles(.npmrc) - 520.2µs
[TRACE] 18:02:08.884 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.884 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.884 shell.go:HasFiles(build.boot) - 1.0382ms
[TRACE] 18:02:08.884 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.884 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.884 shell.go:HasFiles(.vue) - 520.8µs
[DEBUG] 18:02:08.884 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.884 shell.go:HasFiles(*.java) - 520.8µs
[TRACE] 18:02:08.884 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.885 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.885 shell.go:HasFiles(*.class) - 515.8µs
[TRACE] 18:02:08.885 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.885 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.885 shell.go:HasFiles(*.gradle) - 0s
[TRACE] 18:02:08.885 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.886 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.886 shell.go:HasFiles(*.jar) - 519.9µs
[TRACE] 18:02:08.886 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.886 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.886 shell.go:HasFiles(*.clj) - 0s
[TRACE] 18:02:08.886 shell.go:Pwd() - 0s
[DEBUG] 18:02:08.886 shell.go:HasFiles:464 → false
[TRACE] 18:02:08.886 shell.go:HasFiles(*.cljc) - 0s
[TRACE] 18:02:08.887 shell.go:TemplateCache() - 0s
[TRACE] 18:02:08.900 shell_windows_nix.go:BatteryState() - 20.4519ms
[DEBUG] 18:02:08.900 properties.go:GetString:28 → 
[TRACE] 18:02:08.900 shell.go:TemplateCache() - 0s
[DEBUG] 18:02:08.930 shell.go:CommandPath:614 → C:\Users\26523\.cargo\bin\rustc.exe
[TRACE] 18:02:08.930 shell.go:CommandPath(rustc) - 47.0435ms
[TRACE] 18:02:08.930 shell.go:HasCommand(rustc) - 47.0435ms
[DEBUG] 18:02:08.932 shell.go:CommandPath:614 → C:\Users\26523\AppData\Local\Programs\Python\Python311\python.exe
[TRACE] 18:02:08.932 shell.go:CommandPath(python) - 51.857ms
[DEBUG] 18:02:08.932 shell.go:HasFilesInDir:478 → false
[TRACE] 18:02:08.932 shell.go:HasFilesInDir(pyvenv.cfg) - 531.4µs
[DEBUG] 18:02:08.932 shell.go:HasFilesInDir:478 → false
[TRACE] 18:02:08.932 shell.go:HasFilesInDir(pyvenv.cfg) - 0s
[DEBUG] 18:02:08.932 shell.go:Getenv:398 → NO DATA
[TRACE] 18:02:08.932 shell.go:Getenv(VIRTUAL_ENV) - 0s
[DEBUG] 18:02:08.932 shell.go:Getenv:398 → NO DATA
[TRACE] 18:02:08.932 shell.go:Getenv(CONDA_ENV_PATH) - 0s
[DEBUG] 18:02:08.932 shell.go:Getenv:398 → NO DATA
[TRACE] 18:02:08.932 shell.go:Getenv(CONDA_DEFAULT_ENV) - 0s
[DEBUG] 18:02:09.004 shell.go:RunCommand:593 → rustc 1.73.0 (cc66ad468 2023-10-03)
[TRACE] 18:02:09.004 shell.go:RunCommand(rustc --version) - 74.4698ms
[DEBUG] 18:02:09.004 properties.go:GetString:28 → NO DATA
[TRACE] 18:02:09.004 shell.go:TemplateCache() - 0s
[DEBUG] 18:02:09.004 text.go:Render:70 → Rendering template:   {{ (div ((sub .PhysicalTotalMemory .PhysicalFreeMemory)|float64) 1000000000.0) }}/{{ (div .PhysicalTotalMemory 1000000000.0) }}GB
[DEBUG] 18:02:09.004 text.go:Render:70 → Rendering template:  {{ if not .Error }}{{ .Icon }}{{ .Percentage }}{{ end }}
[DEBUG] 18:02:09.004 text.go:Render:70 → Rendering template:   {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }}
[DEBUG] 18:02:09.004 text.go:Render:70 → Rendering template:  {{ if eq .Type "wifi"}} wifi{{ else if eq .Type "ethernet"}} ethernet{{ end }}
[TRACE] 18:02:09.005 shell.go:TemplateCache() - 0s
[TRACE] 18:02:09.005 shell.go:TemplateCache() - 0s
[TRACE] 18:02:09.005 shell.go:TemplateCache() - 0s
[TRACE] 18:02:09.005 shell.go:TemplateCache() - 0s
[TRACE] 18:02:09.005 shell.go:Shell() - 0s
[TRACE] 18:02:09.005 shell.go:Shell() - 0s
[TRACE] 18:02:09.005 shell.go:Shell() - 0s
[TRACE] 18:02:09.005 shell.go:Shell() - 0s
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: {{if eq "Charging" .State.String}}#40c4ff{{end}}
[TRACE] 18:02:09.006 shell.go:TemplateCache() - 0s
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: {{if eq "Discharging" .State.String}}#ff5722{{end}}
[TRACE] 18:02:09.006 shell.go:TemplateCache() - 0s
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: {{if eq "Full" .State.String}}#4caf50{{end}}
[TRACE] 18:02:09.006 shell.go:TemplateCache() - 0s
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: plain
[DEBUG] 18:02:09.006 text.go:Render:70 → Rendering template: plain
[TRACE] 18:02:09.006 shell.go:Shell() - 0s
[DEBUG] 18:02:09.007 shell_windows.go:TerminalWidth:109 → terminal width: 122
[TRACE] 18:02:09.007 shell_windows.go:TerminalWidth() - 581.1µs
[TRACE] 18:02:09.007 shell.go:Shell() - 0s
[TRACE] 18:02:09.007 shell.go:Shell() - 0s
[TRACE] 18:02:09.007 shell.go:Shell() - 0s
[TRACE] 18:02:09.007 shell.go:Flags() - 0s
[TRACE] 18:02:09.007 shell.go:Flags() - 0s
[TRACE] 18:02:09.007 shell.go:Pwd() - 0s
[TRACE] 18:02:09.007 shell.go:Pwd() - 0s
[DEBUG] 18:02:09.007 shell_windows.go:56 → C:\Users\26523
[DEBUG] 18:02:09.007 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:09.007 shell.go:GOOS() - 0s
[TRACE] 18:02:09.007 shell.go:GOOS() - 0s
[DEBUG] 18:02:09.007 segment.go:SetEnabled:512 → Segment: Spotify
[DEBUG] 18:02:09.007 properties.go:GetString:28 → http://127.0.0.1:9863
[DEBUG] 18:02:09.007 properties.go:GetInt:40 → api_url: 20
[DEBUG] 18:02:09.007 shell.go:HTTPRequest:704 ↓
    GET /query HTTP/1.1
    Host: 127.0.0.1:9863
    User-Agent: Go-http-client/1.1
    Accept-Encoding: gzip


[ERROR] 18:02:09.008 shell.go:HTTPRequest:708 → Get "http://127.0.0.1:9863/query": dial tcp 127.0.0.1:9863: connectex: No connection could be made because the target machine actively refused it.
[TRACE] 18:02:09.008 shell.go:HTTPRequest(http://127.0.0.1:9863/query) - 1.6049ms
[ERROR] 18:02:09.011 shell_windows.go:QueryWindowTitles:73 ↓
    no matching window title found

[TRACE] 18:02:09.011 shell_windows.go:QueryWindowTitles(^(Spotify.*)|(.*\s-\s.*)$) - 3.7478ms
[ERROR] 18:02:09.013 shell_windows.go:QueryWindowTitles:73 ↓
    no matching window title found

[TRACE] 18:02:09.013 shell_windows.go:QueryWindowTitles(^(Spotify.*)) - 2.636ms
[TRACE] 18:02:09.013 shell.go:Shell() - 0s
[TRACE] 18:02:09.013 shell.go:Shell() - 0s
[TRACE] 18:02:09.013 shell.go:Shell() - 0s
[TRACE] 18:02:09.013 shell.go:Flags() - 0s
[DEBUG] 18:02:09.013 properties.go:GetString:28 → {{ .Code }}
[TRACE] 18:02:09.013 shell.go:Pwd() - 0s
[DEBUG] 18:02:09.013 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:09.013 shell.go:GOOS() - 0s
[DEBUG] 18:02:09.013 segment.go:SetEnabled:512 → Segment: Status
[TRACE] 18:02:09.013 shell.go:StatusCodes() - 0s
[DEBUG] 18:02:09.013 text.go:Render:70 → Rendering template: {{ .Code }}
[TRACE] 18:02:09.013 shell.go:Flags() - 0s
[TRACE] 18:02:09.013 shell.go:Pwd() - 0s
[DEBUG] 18:02:09.013 shell_windows.go:56 → C:\Users\26523
[TRACE] 18:02:09.013 shell.go:GOOS() - 0s
[DEBUG] 18:02:09.013 segment.go:SetEnabled:512 → Segment: Executiontime
[DEBUG] 18:02:09.013 properties.go:GetBool:22 → always_enabled: false
[TRACE] 18:02:09.013 shell.go:ExecutionTime() - 0s
[DEBUG] 18:02:09.013 properties.go:GetFloat64:34 → threshold: 0.000000
[DEBUG] 18:02:09.013 properties.go:GetString:28 → dallas
[TRACE] 18:02:09.013 shell.go:TemplateCache() - 0s
[TRACE] 18:02:09.013 shell.go:TemplateCache() - 0s
[DEBUG] 18:02:09.014 properties.go:GetBool:22 → always_enabled: true
[TRACE] 18:02:09.014 shell.go:TemplateCache() - 0s
[DEBUG] 18:02:09.014 text.go:Render:70 → Rendering template: {{ if .Error }}<#ff5522>x</>{{ else }}<#63F08C>➜</>{{end}}
[DEBUG] 18:02:09.014 text.go:Render:70 → Rendering template: <#ffffff></> {{ .FormattedMs }}s
[TRACE] 18:02:09.014 shell.go:TemplateCache() - 0s
[TRACE] 18:02:09.014 shell.go:Shell() - 0s
[TRACE] 18:02:09.014 shell.go:TemplateCache() - 0s
[TRACE] 18:02:09.014 shell.go:Shell() - 0s
[DEBUG] 18:02:09.014 shell.go:Getenv:398 → NO DATA
[TRACE] 18:02:09.014 shell.go:Getenv(TERM_PROGRAM) - 0s
[DEBUG] 18:02:09.014 text.go:Render:70 → Rendering template: diamond
[DEBUG] 18:02:09.014 text.go:Render:70 → Rendering template: plain
[TRACE] 18:02:09.014 shell.go:Shell() - 0s
[TRACE] 18:02:09.014 shell.go:Shell() - 0s
[TRACE] 18:02:09.014 shell.go:Shell() - 0s
[DEBUG] 18:02:09.014 shell.go:Getenv:398 → C:\Users\26523\AppData\Local
[TRACE] 18:02:09.014 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 18:02:09.014 shell_windows.go:CachePath() - 0s
[TRACE] 18:02:09.014 shell.go:Flags() - 0s
```

--- a/src/engine/image.go
+++ b/src/engine/image.go
@@ -42,8 +42,8 @@ import (
 
 	"github.com/esimov/stackblur-go"
 	"github.com/fogleman/gg"
-	"github.com/golang/freetype/truetype"
 	"golang.org/x/image/font"
+	"golang.org/x/image/font/opentype"
 )
 
 type ConnectionError struct {
@@ -217,7 +217,7 @@ func (ir *ImageRenderer) loadFonts() error {
 
 	// Download font if not cached
 	if data == nil {
-		url := "https://github.com/ryanoasis/nerd-fonts/releases/download/v3.1.0/Hack.zip"
+		url := "https://github.com/ryanoasis/nerd-fonts/releases/download/v3.2.1/Hack.zip"
 		var err error
 
 		data, err = fontCLI.Download(url)
@@ -237,7 +237,7 @@ func (ir *ImageRenderer) loadFonts() error {
 		return err
 	}
 
-	fontFaceOptions := &truetype.Options{Size: 2.0 * 12, DPI: 144}
+	fontFaceOptions := &opentype.FaceOptions{Size: 2.0 * 12, DPI: 144}
 
 	parseFont := func(file *zip.File) (font.Face, error) {
 		rc, err := file.Open()
@@ -252,12 +252,16 @@ func (ir *ImageRenderer) loadFonts() error {
 			return nil, err
 		}
 
-		font, err := truetype.Parse(data)
+		font, err := opentype.Parse(data)
 		if err != nil {
 			return nil, err
 		}
 
-		return truetype.NewFace(font, fontFaceOptions), nil
+		fontFace, err := opentype.NewFace(font, fontFaceOptions)
+		if err != nil {
+			return nil, err
+		}
+		return fontFace, nil
 	}
 
 	for _, file := range zipReader.File {

--- a/src/go.mod
+++ b/src/go.mod
@@ -10,7 +10,6 @@ require (
 	github.com/alecthomas/repr v0.4.0 // indirect
 	github.com/esimov/stackblur-go v1.1.0
 	github.com/fogleman/gg v1.3.0
-	github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0
 	github.com/google/uuid v1.6.0 // indirect
 	github.com/gookit/color v1.5.4
 	github.com/huandu/xstrings v1.4.0 // indirect
@@ -70,6 +69,7 @@ require (
 	github.com/containerd/console v1.0.4 // indirect
 	github.com/dsnet/compress v0.0.1 // indirect
 	github.com/fatih/color v1.16.0 // indirect
+	github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0 // indirect
 	github.com/google/go-cmp v0.6.0 // indirect
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	github.com/lucasb-eyer/go-colorful v1.2.0 // indirect

